tests/labelled_tests.rs

```diff
@@ -277,3 +278,30 @@ fn test_transmogrify_keyword_field_embedder_structs() {
     let result: HasKeyword2Embedder = value.transmogrify();
     assert_eq!(3, result.r#true.r#type);
 }
+#[test]
+fn test_pluck_by_name() {
+    let u = NewUser {
+        first_name: "Humpty",
+        last_name: "Drumpty",
+        age: 3,
+    };
+    let h = into_labelled_generic(u);
+    let first_name: (Field<(f, i, r, s, t, __, n, a, m, e), _>, _) = h.pluck_by_name();
+    assert_eq!(first_name.0.value, "Humpty");
+    let age: (Field<(a, g, e), _>, _) = h.pluck_by_name();
+    assert_eq!(age.0.value, 3);
+}
+
+#[test]
+fn test_pluck_by_name_ref() {
+    let u = NewUser {
+        first_name: "Humpty",
+        last_name: "Drumpty",
+        age: 3,
+    };
+    let h = &into_labelled_generic(u);
+    let first_name: (Field<(f, i, r, s, t, __, n, a, m, e), _>, _) = h.pluck_by_name();
+    assert_eq!(first_name.0.value, &"Humpty");
+    let age: (Field<(a, g, e), _>, _) = h.pluck_by_name();
+    assert_eq!(age.0.value, &3);
+}
```

Comment:
I think having it here in `tests` is great but it might be good to also have (unit?) testing without using the `into_labelled_generic` machinery like so

https://github.com/lloydmeta/frunk/blob/ffba8cfa144b8ed02942271a3ea45006100cd029/core/src/labelled.rs#L972-L977